.ci/azure/linux_onnxruntime.yml

```diff
@@ -91,6 +93,19 @@ jobs:
   - script: |
       set -e
       $(REPO_DIR)/install_build_dependencies.sh
+      wget https://www.python.org/ftp/python/"$PYTHON_VERSION"/Python-"$PYTHON_VERSION".tgz
+      tar -xzf Python-"$PYTHON_VERSION".tgz
+      cd Python-"$PYTHON_VERSION" || exit
+      ./configure --enable-optimizations --prefix=/usr/local --enable-shared LDFLAGS="-Wl,-rpath /usr/local/lib"
+      make python
```

Comment:
here and maybe in other places we are just wasting machine time with building python each time. 
I still suppose we need to enable python 3.10 only on Jenkins, while on Azure we should use default python version for the system (for Windows we can install 3.10 since Windows does not have python by default)

Reply:
Answered in an internal channel.